### PR TITLE
osa - add metricsbridge image to the templates

### PR DESCRIPTION
**Pull request by mjudeikis**
Adds metricsbridge for e2e tests.

### CODE REVIEW

**Comment by mjudeikis** [comment]
> I think you are also missing the --target flags in the jobs that use the templates?

I always forget that one :/

**Comment by kargakis** [comment]
Needs a rebase, also I think you will need to sort the targets alpabetically to please one of the jobs.

**Comment by mjudeikis** [comment]
@kargakis fixed

File: ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml [hunk 1]
```diff
@@ -180,6 +182,8 @@ objects:
         value: /tmp/shared/home
       - name: RESOURCEGROUP
         value: e2e-${RESOURCEGROUP_SUFFIX}
+      - name: METRICSBRIDGE_IMAGE
```
Comment (kargakis) [inline ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml]: Mind sorting this?

File: ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml [hunk 2]
```diff
@@ -14,6 +14,8 @@ parameters:
   required: true
 - name: IMAGE_AZURE_CONTROLLERS
   required: true
+- name: IMAGE_METRICSBRIDGE
+  required: true
```
Comment (kargakis) [inline ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml]: Why are these required? Shouldn't we default to a value if none is passed?

Comment (mjudeikis) [reply ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml]: Required because image is built with each PR, and we want to test with it. I better make it mandatory than miss some bug.

Comment (kargakis) [reply ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml]: How is this going to affect us running the template locally?

File: ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml [hunk 3]
```diff
@@ -190,6 +193,9 @@ objects:
         value: 4h
       - name: RUNNING_UNDER_TEST
         value: ${RUNNING_UNDER_TEST}
+        value: "true"
```
Comment (kargakis) [inline ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml]: looks like a bad rebase

File: ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml [hunk 4]
```diff
@@ -190,6 +193,9 @@ objects:
...
+      - name: METRICSBRIDGE_IMAGE
```
Comment (kargakis) [inline ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml]: Please keep these sorted

File: ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml [hunk 5]
```diff
@@ -18,6 +18,9 @@ parameters:
 - name: IMAGE_SYNC
   required: true
 - name: LOCAL_IMAGE_BIN
+- name: IMAGE_METRICSBRIDGE
```
Comment (kargakis) [inline ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml]: Please keep these sorted

**Comment by kargakis** [comment]
You seem to be missing two new presubmits and one periodic
```
ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-presubmits.yaml
22:        - --target=[output:stable:sync]
81:        - --target=[output:stable:sync]
144:        - --target=[output:stable:sync]
207:        - --target=[output:stable:sync]
266:        - --target=[output:stable:sync]
331:        - --target=[output:stable:sync]

ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml
212:      - --target=[output:stable:sync]
266:      - --target=[output:stable:sync]
320:      - --target=[output:stable:sync]
511:      - --target=[output:stable:sync]
```

File: ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml (hunk 2 above)
Comment (mjudeikis) [reply ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml]: It should not. otherwise, we would already see "the issue" with `IMAGE_SYNC` and `IMAGE_AZURE_CONTROLLERS` as they use the same pattern.

**Comment by kargakis** [comment]
Closing in favor of https://github.com/openshift/release/pull/2347
